compiler/src/iree/compiler/Codegen/Common/FoldTensorExtractOp.td

```diff
@@ -17,6 +17,6 @@ include "mlir/IR/PatternBase.td"
 def : Pat<(Tensor_ExtractOp(Bufferization_ToTensorOp $value, $restrict,
                             $writable),
            $indices),
-          (LoadOp $value, $indices, ConstBoolAttrFalse)>;
+          (LoadOp $value, $indices, ConstBoolAttrFalse, ConstantAttr<I32Attr, "0">)>;
```

Comment:
That might want to be a native code call producing the null attribute to make it optional?